xclim/indicators/generic/_stats.py

```diff
@@ -44,11 +44,11 @@ class GenericResampling(ResamplingIndicator):
 
 
 stats = GenericResampling(
-    title="Statistic of the daily values for a given period.",
+    title="Simple resampled statistic of the values.",
```

Comment:
Does this need to be updated in the French translation?

Reply:
Indeed.